examples/notebooks/use-cases/ResolvingStrategies.ipynb

```diff
@@ -147,18 +155,144 @@
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "Resource(_last_action=None, _validated=False, _synchronized=False, _store_metadata=None, id='https://bbp.epfl.ch/ontologies/core/ttypes/264_L5_6_NP_CTX', type=['Entity', 'Class'], label='264_L5/6 NP CTX', _inner_sync=False, subClassOf=['https://bbp.epfl.ch/ontologies/core/ttypes/L5_6_NP_CTX', 'https://bbp.epfl.ch/ontologies/core/celltypes/NeuronTranscriptomicType'])"
-      ]
-     },
-     "execution_count": 11,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "    id: https://bbp.epfl.ch/ontologies/core/ttypes/264_L5_6_NP_CTX\n",
+      "    type:\n",
+      "    [\n",
+      "        Entity\n",
+      "        Class\n",
+      "    ]\n",
+      "    label: 264_L5/6 NP CTX\n",
+      "    subClassOf:\n",
+      "    [\n",
+      "        https://bbp.epfl.ch/ontologies/core/ttypes/L5_6_NP_CTX\n",
+      "        https://bbp.epfl.ch/ontologies/core/celltypes/NeuronTranscriptomicType\n",
+      "    ]\n",
+      "}\n"
+     ]
     }
    ],
    "source": [
-    "forge.resolve('264_L5/6 np CTX', scope='ontology', strategy='EXACT_CASEINSENSITIVE_MATCH')"
+    "print(forge.resolve('264_L5/6 np CTX', scope='ontology', strategy='EXACT_CASEINSENSITIVE_MATCH'))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Select which ontology to point when resolving\n",
+    "This is particularly usefull when two entities have a similar acronym like the example below, where there is a brain region called BAC, and a electrical cell type labelled bAC"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Without any specification, it will return the first result (no priority or ranking)"
```

Comment:
"first" in which set?
I would replace "the first result" with something like "a random element of the matched results".

Reply:
OK